nautobot-app/{{ cookiecutter.project_slug }}/docs/dev/release_checklist.md

```diff
@@ -0,0 +1,210 @@
+# Release Checklist
+
+This document is intended for app maintainers and covers the steps to perform when releasing new versions of the app.
+
+!!! important
+    Before starting, make sure your **local** `develop`, `main`, and (if applicable) the current LTM branch are all up to date with upstream!
+
+    ```
+    git fetch
+    git switch develop && git pull # and repeat for main/ltm
+    ```
+
+Choose your own adventure:
+
+- LTM release? Jump [here](#ltm-releases).
+- Patch release from `develop`? Jump [here](#all-releases-from-develop).
+- Minor release? Continue with [Minor Version Bumps](#minor-version-bumps) and then [All Releases from `develop`](#all-releases-from-develop).
+
+## Minor Version Bumps
+
+### Update Requirements
+
+Every minor version release should refresh `poetry.lock`, so that it lists the most recent stable release of each package. To do this:
+
+0. Run `poetry update --dry-run` to have Poetry automatically tell you what package updates are available and the versions it would upgrade to. This requires an existing environment created from the lock file (i.e. via `poetry install`).
```

Comment:
0 indexed for docs? Is this an easter egg? :)

Reply:
Only in markdown, the html conversion uses the "normie" index ;) https://docs.nautobot.com/projects/dev-example/en/latest/dev/release_checklist/#minor-version-bumps